x/concentrated-liquidity/incentives_test.go

```diff
@@ -2378,53 +2362,54 @@ func (s *KeeperTestSuite) TestQueryAndCollectIncentives() {
 				validPoolId := validPool.GetId()
 
 				fmt.Println("expected incentives claimed for: ", name, " ", tc.expectedIncentivesClaimed.String())
-				s.FundAcc(validPool.GetIncentivesAddress(), tc.expectedIncentivesClaimed)
+
+				// Fund the incentives address with amount we intend to claim and forfeit.
+				s.FundAcc(validPool.GetIncentivesAddress(), tc.expectedIncentivesClaimed.Add(tc.expectedForfeitedIncentives...))
```

Comment:
Note to reviewer: we now need to fund the incentives account with both the incentives and forfeited amount, since two bank sends happen in CollectIncentives